app-catalog/src/components/charts/List.tsx

```diff
@@ -166,150 +204,177 @@ export function ChartsList({ fetchCharts = fetchChartsFromArtifact }) {
-                      style={{
-                        width: '60px',
-                        margin: '1rem',
-                        alignSelf: 'flex-start',
-                      }}
-                      component="img"
-                    />
+          <Box
+            display="flex"
+            m={1}
+            justifyContent="center"
+            flexWrap="wrap"
+            sx={{
+              flexDirection: {
+                xs: 'column',
+                sm: 'row',
+              },
+            }}
+          >
+            {charts.map(chart => {
+              return (
+                <Box
+                  m={1}
+                  sx={{
+                    width: {
+                      xs: '100%',
+                      sm: '100%',
+                      md: '30%',
+                    },
+                    maxWidth: {
+                      xs: '100%',
+                      sm: '100%',
+                      md: '30%',
+                    },
+                  }}
+                >
+                  <Card>
                     <Box
+                      height="60px"
                       display="flex"
                       alignItems="center"
-                      justifyContent="space-around"
-                      marginRight="10px"
+                      justifyContent="space-between"
+                      marginTop="15px"
                     >
-                      {(chart.cncf || chart.repository.cncf) && (
-                        <Tooltip title="CNCF Project">
-                          <Icon
-                            icon="simple-icons:cncf"
-                            style={{
-                              marginLeft: '0.5em',
-                              fontSize: '20px',
-                            }}
-                          />
-                        </Tooltip>
-                      )}
-                      {(chart.official || chart.repository.official) && (
-                        <Tooltip title="Official Chart">
-                          <Icon
-                            icon="mdi:star-circle"
-                            style={{
-                              marginLeft: '0.5em',
-                              fontSize: '22px',
-                            }}
-                          />
-                        </Tooltip>
-                      )}
-                      {chart.repository.verified_publisher && (
-                        <Tooltip title="Verified Publisher">
-                          <Icon
-                            icon="mdi:check-decagram"
-                            style={{
-                              marginLeft: '0.5em',
-                              fontSize: '22px',
-                            }}
-                          />
-                        </Tooltip>
-                      )}
+                      <CardMedia
+                        image={`https://artifacthub.io/image/${chart.logo_image_id}`}
```

Comment:
This CardMedia is missing an alt attribute description for a11y.

Reply:
done!